recipe/meta.yaml

```
@@ -31,18 +37,34 @@ requirements:
     - typing-extensions
 
 test:
+  source_files:
+    - tests
   imports:
     - google
   commands:
     - pip check
+    - pytest -v tests
   requires:
     - pip
+    - pytest
+    - absl-py
+    - grpcio
+    - black
+    - nose2
```

```suggestion
    - black
    - nose2
```
You are using pytest instead of nose2. I don't think nose2 is necessary. I doubt black is used for testing too.